exercise-2/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Exercise2</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;600&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <h2
+      style="
```

Comment:
why did we assign style attributes here (using inline CSS) when we have the CSS file handy?

Reply:
Sir i thought this will be useful for those element who have just 1-2 styling properties , in this way i can also reduced the no of classes . Will take care from next time